source/_integrations/growatt_server.markdown

```diff
@@ -54,7 +55,16 @@ Using an API token is recommended for MIN/TLX inverters as it uses the official
 
 When using API token authentication with MIN/TLX inverters, the integration provides additional control entities:
 
-### Switch entities
+{% important %}
+These controls directly modify your inverter's operational settings. Only change these values if you understand their impact on your system. Incorrect settings may damage your battery, reduce system efficiency, or void your warranty. Use at your own risk.
+{% endimportant %}
 
-- **AC charge**: Enable or disable AC charging
+The following control entities are available:
 
+| Name                | Type   | Unit | Description   |
+|---------------------|--------|------|:--------------|
```

Comment:
We prefer lists over tables.
https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/_integration_docs_template.markdown?plain=1#numbers
https://developers.home-assistant.io/docs/documenting/general-style-guide#tables

Example:
- **Charge power** (%). Number. Set the charge power as a percentage (0-100%).